install_config/web_console_customization.adoc

```diff
@@ -376,6 +376,41 @@ assetConfig:
 ----
 ====
 
+. As a cluster administrator, run the following command to give users the
```

Comment:
@jwforres @bparees This was sourced from https://github.com/nickschuetz/ocp-3.3-pipelines/blob/master/README.md. However, it is noted that the steps arose from a possible edge case during an upgrade from 3.2 to 3.3, so this content may not be valid.